### PR TITLE
Add example for :class:~.`ParametricSurface`

### PR DESCRIPTION
**Pull request by Nikhil0504**
## Changelog / Overview

Add new example for parametric function.

## Motivation

Makes the docs better.
## Explanation for Changes

[Documentation Reference](https://manimce--####.org.readthedocs.build)

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Comment by hydrobeam** [comment]
Shouldn't this be in ParametricSurface instead 🤔

**Comment by Nikhil0504** [comment]
> Shouldn't this be in ParametricSurface instead

Oh yeah 😅 . I'll change it

File: manim/mobject/three_dimensions.py [hunk 1]
```diff
@@ -32,6 +32,24 @@ def __init__(self, shade_in_3d=True, **kwargs):
...
+    .. manim:: ParaSurface
+        :save_last_frame:
+
+        class ParaSurface(ThreeDScene):
+            def func(self, u, v):
+                return np.array([np.cos(u) * np.cos(v), np.cos(u) * np.sin(v), u])
+
+            def construct(self):
+                axes = ThreeDAxes()
+                surface = ParametricSurface(lambda u, v: self.func(u, v), u_min=-TAU, u_max=TAU / 2, v_min=0, v_max=2 * PI)
```
Comment (jsonvillanueva) [inline manim/mobject/three_dimensions.py:48]: The lambda is unnecessary here:
```suggestion
                surface = ParametricSurface(self.func, u_min=-TAU, u_max=TAU / 2, v_min=0, v_max=2 * PI)
```

**Review by jsonvillanueva — CHANGES_REQUESTED** [review]
The example looks fine to me otherwise

File: manim/mobject/three_dimensions.py [hunk 2]
```diff
@@ -32,6 +32,24 @@ def __init__(self, shade_in_3d=True, **kwargs):
...
+    --------
+    .. manim:: ParaSurface
+        :save_last_frame:
+
+        class ParaSurface(ThreeDScene):
+            def func(self, u, v):
+                return np.array([np.cos(u) * np.cos(v), np.cos(u) * np.sin(v), u])
+
+            def construct(self):
+                axes = ThreeDAxes()
```
Comment (abhi3019) [inline manim/mobject/three_dimensions.py:47]: I feel like the current x-axis range - (-6, 6) is way too big. Not really an issue, but it looks good to me when the range and length are adjusted to - `ThreeDAxes(x_range=[-4,4], x_length=8)`. 

The final render together with the suggestion below looks like this: 
ParaSurface_ManimCE_v0 7 0

**Review by abhi3019 — CHANGES_REQUESTED** [review]

File: manim/mobject/three_dimensions.py (hunk 2 above)
Comment (Nikhil0504) [reply manim/mobject/three_dimensions.py]: I took the suggestions into consideration. Thank you.

**Review by jsonvillanueva — APPROVED** [review]

**Pull request merged**